db-demo/processes/playerDemo.p.json

```diff
@@ -0,0 +1,92 @@
+{
+  "format" : "10.0.0",
+  "id" : "184BD5B00AD6A83D",
+  "config" : {
+    "data" : "db.demos.Player"
+  },
+  "elements" : [ {
+      "id" : "f0",
+      "type" : "RequestStart",
+      "name" : "start.ivp",
+      "config" : {
+        "callSignature" : "start",
+        "outLink" : "start.ivp"
+      },
+      "visual" : {
+        "at" : { "x" : 280, "y" : 240 }
+      },
+      "connect" : { "id" : "f2", "to" : "f3" }
+    }, {
+      "id" : "f1",
+      "type" : "TaskEnd",
+      "visual" : {
+        "at" : { "x" : 744, "y" : 288 }
+      }
+    }, {
+      "id" : "f3",
+      "type" : "Database",
+      "name" : "create",
+      "config" : {
+        "query" : {
+          "kind" : "any",
+          "stmt" : [
+            "create table players(",
+            "name varchar(200),",
+            "birthday date,",
+            "favoritecolor varchar(50)",
+            ")"
+          ]
+        },
+        "dbUrl" : "FootballDB"
+      },
+      "visual" : {
+        "at" : { "x" : 512, "y" : 240 }
+      },
+      "connect" : { "id" : "f4", "to" : "f1", "via" : [ { "x" : 744, "y" : 240 } ] }
+    }, {
+      "id" : "f6",
+      "type" : "RequestStart",
+      "name" : "start2.ivp",
```

Comment:
this is a public demo: please use an expressive name, describing the purpose of this start
e.g. crudPlayerDialog